src/parser/jobs/nin/modules/Positionals.ts

```diff
@@ -21,11 +30,11 @@ export class Positionals extends CorePositionals {
 		action: this.data.actions.AEOLIAN_EDGE,
 		potencies: [
 			{
-				value: 140,
+				value: AEOLIAN_EDGE_BASE_POTENCY,
```

Comment:
Same here re base potency.

Reply:
Resolved in [95e3207](https://github.com/xivanalysis/xivanalysis/pull/1704/commits/95e3207223d677ad20b684a6d8e0d21b6e397510)

Reply:
This too still isn't resolved.